src/languages/lib/css-shared.js

```diff
@@ -1,3 +1,137 @@
+export const MODES = (hljs) => {
```

Comment:
Doesn't `MODES` sound too broad for these 3 modes? You'll have to guess what `  const modes = css.MODES(hljs);` really means.

Reply:
Seems clear to me?   MODES just for CSS vs MODES in general.

Not sure I follow.  Did you have an alternative suggestion that you think is clearer?

I wrapped them so I could pass in `hljs`... seems silly to have a 3+ separate functions and have to pass them all hljs.

Reply:
No, I don't have anything else but 3 separate functions. OK if single function seems clear enough to you.